tests/acceptance/features/apiCors/cors.feature

```diff
@@ -0,0 +1,70 @@
+#
+# NOTE: for running this feature locally, you need to run oCIS with the following env var:
+# CORS_ALLOWED_ORIGINS=https://aphno.badal
+#
```

Comment:
```suggestion
# NOTE: for running this feature locally, you need to run oCIS with the following env var:
# CORS_ALLOWED_ORIGINS=https://aphno.badal
```
Why these extra `#` ?

Reply:
just like you notice it, I want others to notice the odd and read the NOTE :smile: